### ISSUE TITLE
Potentially confusing note in definition of *Part* structure element

### ISSUE THREAD

**Issue by MatthiasValvekens** [pdf-association/pdf-issues#141]
This one has been bothering me for a while, so I thought I'd ask here.

Table 365 (Grouping level structure types) in ISO 32000-2:2020 has the following to say about **Part**:

> Encloses a grouping of structure elements without consideration for their hierarchy.
> NOTE 1: **Part** is the semantic equivalent of **Div**.
> (rest of table cell omitted)

I think there are several ways to read that note:

 - **Part** is semantically equivalent to **Div** (this interpretation clearly doesn't make sense, otherwise there's no point in having two separate SEs)
 - **Part** is like **Div** applied to semantics[, as opposed to something else?] (layout?)
 - **Part** is like **Div** [in that it is non-hierarchical], but unlike **Div** it has semantic [grouping] value[, whereas **Div** is just a vehicle for attributes or inserted for layout convenience]

I believe the 3rd reading is the intended one, but I end up doing a double take every time I read that note, and the interpretation of **Part** comes up in meetings occasionally. My question is twofold:

 - Is my reading consistent with the intention of the note?
 - Can we rephrase the note to make said intention more obvious to the reader?

Tagging this as a question rather than a bug for now.

**Comment by petervwyatt** [comment]
Yes we can rephrase the note safely, since notes are only informative and not normative.
And I would agree with your interpretation 3.

**Comment by lrosenthol** [comment]
Agreed that it is the 3rd bullet.  But @mrbhardy should weigh in here too...

**Comment by MatthiasValvekens** [comment]
Here's an initial proposal to rewrite the note:

> NOTE 1: The non-hierarchical aspect of **Part** is similar to **Div**. However, unlike **Div**, the grouping of elements enclosed in a **Part** structure element has semantic value.

To acknowledge the point made by @mrbhardy in the previous PDF TWG meeting: I made an attempt to tweak my wording to avoid suggesting that "the semantic value of **Part** _is_ grouping" (which isn't fully accurate), but rather that the grouping indicated by **Part** is of a semantic nature. If my recollection of what we said at the PDF TWG is correct, the _full_ semantic value of the **Part** SE is defined by its children _together_ with that grouping aspect.

I didn't put that last nuance in my reworded version of the note, because the table cell I quoted in my original post goes on to say the following:

> A structure element with the type of **Part** shall inherit the containment requirements and limitations of its parent element. Where the parent element is itself a structure element of type **Part**, then the inheritance shall recurse to the first parent element whose type is not **Part**.

I feel that any additional clarification on what **Part**'s semantic value is would be better off in a second note after that paragraph. That would keep Note 1 on topic (i.e. comparing **Part** <> **Div**). Such a note could then read something like:

> NOTE 2: The semantic value of a structure element of type **Part** is determined by the elements enclosed within, in addition to the grouping nature of **Part**.

Would such a note be worthwhile? Necessary? Actively harmful? Just plain wrong? 
(Bookkeeping remark: adding an extra note would require renumbering other notes further down in 14.8.4.4)

I'll hold off on marking this as "proposed solution" to gather some additional feedback, and to allow time for wordsmithing/deciding whether we need that second note.
